src/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableDictionary_2.MutationInput.cs

```diff
@@ -98,14 +98,6 @@ internal IEqualityComparer<HashBucket> HashBucketComparer
             {
                 get { return _comparers.HashBucketEqualityComparer; }
             }
-
-            /// <summary>
-            /// Gets the current number of elements in the collection.
-            /// </summary>
-            internal int Count
-            {
-                get { return _count; }
```

Comment:
Is the `_count` field used anywhere else? Should it be deleted too?

Reply:
Good catch. I'll delete.